src/lambdas/archive-user-logs/src/types/index.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod"
```

Comment:
Can we please move the types from this file to their own files and export them here? Similar to other types in audit logging.
i.e.

index.ts:
```
export { default as ValidLogEvent } from "./ValidLogEvent"
```

ValidLogEvent.ts:
```
const ValidLogEvent = z.object({
  id: z.string(),
  timestamp: z.number(),
  message: z.string()
})

export default ValidLogEvent
```